DirectProgramming/DPC++FPGA/ReferenceDesigns/anr/src/CMakeLists.txt

```diff
@@ -4,13 +4,13 @@ set(EMULATOR_TARGET ${TARGET_NAME}.fpga_emu)
 set(FPGA_TARGET ${TARGET_NAME}.fpga)
 
 # FPGA board selection
-if(NOT DEFINED FPGA_BOARD)
-    set(FPGA_BOARD "intel_a10gx_pac:pac_a10")
-    message(STATUS "FPGA_BOARD was not specified.\
-                    \nConfiguring the design to run on the default FPGA board ${FPGA_BOARD} (Intel(R) PAC with Intel Arria(R) 10 GX FPGA). \
+if(NOT DEFINED FPGA_TARGET)
```

Comment:
I noticed this in Mark's PR as well. I think this conflicts with the FPGA_TARGET variable set a few lines above. Maybe the variable above can be changed to HARDWARE_TARGET instead?